src/main/java/hyperledger/besu/java/rest/client/controller/EthClientController.java

```diff
@@ -19,10 +22,10 @@ public class EthClientController {
   /** */
   @PostMapping(value = "/transactions")
   public ResponseEntity<ClientResponseModel> executeTransaction(
-      @RequestParam("abi-definition-file") MultipartFile abiDefinitionFile,
+      @RequestParam("abi-definition-file") @Validated MultipartFile abiDefinitionFile,
```

Comment:
Suggestion : We can rename it to "abi-definition"

Reply:
updated